baremetal/metal3machine_manager_test.go

```diff
@@ -1186,6 +1186,37 @@ var _ = Describe("Metal3Machine manager", func() {
 				ExpectSecretDeleted: true,
 			},
 		),
+		Entry("Consumer ref should be removed for StateAvailable", testCaseDelete{
+			Host: newBareMetalHost("myhost", bmhSpecNoImg(), bmh.StateAvailable,
+				bmhStatus(), false, false,
+			),
+			Machine: newMachine("mymachine", "", nil),
+			BMMachine: newMetal3Machine("mybmmachine", nil, bmmSecret(), nil,
+				bmmObjectMetaWithValidAnnotations(),
+			),
+			Secret:              newSecret(),
+			ExpectSecretDeleted: true,
+		}),
+		Entry("Consumer ref should be removed for StateAvailable", testCaseDelete{
```

Comment:
Can you please make those entries with unique names ? you now have two `Consumer ref should be removed for StateAvailable`